components/Signin.tsx

```diff
@@ -61,45 +50,32 @@ const SignIn = () => {
       if (isSignInWithEmailLink(auth, window.location.href)) {
         let email = window.localStorage.getItem("emailForSignIn");
 
-        if (!email) {
-          email = window.prompt("Please enter your email to complete sign-in:");
-        }
-
-        if (!email) {
-          setError("Email is required to complete sign-in.");
+        if (!email && !(email = window.prompt("Please enter your email to complete sign-in:"))) {
```

Comment:
```suggestion
        if (!email) {
```
just this much is enough as we are directly handing with toast now :)

Reply:
Sure bhaiya, i will remove it.

Reply:
Sure